omnigan/trainer.py

```diff
@@ -856,6 +879,70 @@ def get_d_loss(self, multi_domain_batch, verbose=0):
         loss = sum(v for d in disc_loss.values() for k, v in d.items())
         return loss
 
+    def update_ADVENT_D(self, multi_domain_batch, verbose=0):
+        # ? split representational as in update_g
+        # ? repr: domain-adaptation traduction
+        self.ADVENT_D_opt.zero_grad()
+        ADVENT_adver_loss = self.get_ADVENT_adver_loss(multi_domain_batch, verbose)
```

Comment:
usually in python only class names start with capital letters, variables should start with lower case letters ;)